student.rb

```diff
@@ -0,0 +1,12 @@
+class Student < Person
+  attr_accessor :classroom
+
+  def initialize(classroom)
+    super()
```

Comment:
- [ ] I suggest that you pass the arguments when calling super() method. This is because in the Person class the initialize expects at least 2 arguments . Therefore it would be mandatory to pass `age` in the Student initialize and super() method and additionally pass other arguments(name, parent_permission) ☺️.

Reply:
ok thx